omnibus/files/private-chef-cookbooks/private-chef/templates/default/sv-bookshelf-run.erb

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 exec 2>&1
 
+export PATH=/opt/opscode/embedded/bin:$PATH
```

Comment:
Since we don't call `env` with `-i` I don't think there is really a difference and that line was getting pretty long.